README.md

```diff
@@ -7,56 +7,48 @@
-Steps below:
-1. On Chrome, go to https://bintangbadminton.org/court-reservations/
-2. Scroll to the bottom and look for this: ![image](https://user-images.githubusercontent.com/10181537/147729891-ad980af9-cf10-40c8-8c0d-d8dae8c91299.png)
-3. `⌥+⌘+I` to open developer tools
-4. On the developer tools, click the "Network" tab: <img width="1792" alt="Screen Shot 2021-12-29 at 23 18 35" src="https://user-images.githubusercontent.com/10181537/147730076-07251ad4-b126-488a-8e78-02c5371d691a.png">
-5. Do `⌘+E` to clear the network history 
-6. Click any court on the left, e.g. Campbell Courts: <img width="1788" alt="Screen Shot 2021-12-29 at 23 20 23" src="https://user-images.githubusercontent.com/10181537/147730150-8647b514-bd99-496d-9cf2-85f811f41258.png">
-7. Some requests should show up on the right: <img width="350" alt="Screen Shot 2021-12-29 at 23 22 44" src="https://user-images.githubusercontent.com/10181537/147730281-0f5755ee-fe73-4307-a3cc-72a8011b7e95.png">
-8. Keep clicking around the list until you see a `Request URL` that starts with `https://app.bukza.com/api/`: <img width="922" alt="Screen Shot 2021-12-29 at 23 24 45" src="https://user-images.githubusercontent.com/10181537/147730444-f1a8f873-5a58-41f3-b422-29f94e0a7c5a.png">
-9. Scroll down to `Request Headers` and copy the long text beside "Bearer". This is your Bearer Token: <img width="782" alt="Screen Shot 2021-12-29 at 23 28 00" src="https://user-images.githubusercontent.com/10181537/147730765-71bd112a-0d8b-4393-b047-5af3d486b677.png">
+1. `-d`: Date string, in the form of `YYYY-MM-DD`
+2. `-g`: The names of the gyms you'd like to load court schedules for. If you leave this empty you'll get every court for every gym, and you'll be waiting for a while. Your choices: `campbell, dublin, milpitas, san carlos, sunnyvale`.
 
+
+# Credits
+This is a fork of [teoh/bintang-buddy](https://github.com/teoh/bintang-buddy) which provided a lot of the base functionality (kudos!). However,
+- [teoh/bintang-buddy](https://github.com/teoh/bintang-buddy) required users to clone the repository, create a virtual environment, activate it, install the project inside the virtual environment, obtain a Bearer token, set an environment variable with the Bearer token and run a Python script.
+- This process was quite involved even for a moderately technical person. 
+
+This fork, however:
+- Creates an easy-to-use CLI based on [click](https://click.palletsprojects.com/en/latest/) which is published to `pypi`. The CLI can be installed through a single command and is ready to go.
+- Removes the need to manually (and painstakingly) obtain a `Bearer` token for authentication (as described [here](https://github.com/teoh/bintang-buddy#getting-the-bearer-token)). This is now done automatically behind the scenes.
+- Uses multithreading to significantly reduce load times.
+- Uses linting and formatting tools to keep code clean and organized.
```

Comment:
For posterity, please add this to your PR description. This is the most detailed description to the great work you've done, and since this section likely won't stick around on the README as-is, I'd like to make sure it's easy for others to see what you've contributed.